test/server/cards/01_SOR/units/ZebOrreliosHeadstrongWarrior.spec.ts

```diff
@@ -28,26 +36,60 @@ describe('Zeb Orrelios, Headstrong Warrior', function () {
                 expect(context.player2).toBeActivePlayer();
 
                 // reset
-                context.zebOrrelios.exhausted = false;
-                context.setDamage(context.zebOrrelios, 0);
                 context.setDamage(context.consularSecurityForce, 0);
-                context.player2.passAction();
+                reset();
 
                 // consular security force is not defeat, nothing happen
                 context.player1.clickCard(context.zebOrrelios);
                 context.player1.clickCard(context.consularSecurityForce);
                 expect(context.player2).toBeActivePlayer();
 
                 // reset
-                context.zebOrrelios.exhausted = false;
-                context.setDamage(context.zebOrrelios, 0);
-                context.player2.passAction();
+                reset(false);
+
+                // consular security force attack and die : nothing happen
+                context.player2.clickCard(context.consularSecurityForce);
+                context.player2.clickCard(context.zebOrrelios);
+                expect(context.player1).toBeActivePlayer();
 
                 // zeb kill someone but die too, nothing happen
                 context.player1.clickCard(context.zebOrrelios);
                 context.player1.clickCard(context.steadfastBattalion);
                 expect(context.player2).toBeActivePlayer();
             });
         });
+
+        describe('Zeb Orrelios\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [{ card: 'zeb-orrelios#headstrong-warrior', upgrades: ['vambrace-flamethrower'] }]
+                    },
+                    player2: {
+                        groundArena: ['battlefield-marine', 'wampa']
+                    }
+                });
+            });
+
+            it('should deal 4 damage to a ground unit when zeb attacks and kill with on attack abilities', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.zebOrrelios);
+                context.player1.clickCard(context.battlefieldMarine);
+
+                // kill battlefield marine with flamethrower, zeb should deal 4 damage to another unit
```

Comment:
Awesome!! Thanks again for adding this and catching that rules issue, very helpful